src/pybind/mgr/ansible/module.py

```diff
@@ -455,6 +569,45 @@ def process_playbook_result(inventory_events, ar_client, playbook_uuid):
 
     return result
 
+def process_hosts_ls(host_ls_json):
+    """ Format the output of host ls call
+
+    Ansible runner service provides the following json:
+            {
+            "status": "OK",
+            "msg": "",
+            "data": {
+                "hosts": [
+                    "192.168.121.20",
+                    "192.168.121.170",
+                    "192.168.121.213",
+                    "192.168.121.212"]
+                }
+            }
+    The output provide is like:
+
+    Hosts
+    192.168.121.20
+    192.168.121.170
+    192.168.121.213
+    192.168.121.212
```

Comment:
would it make sense to just return a list of IPs? And let the caller run `"\n"-join(process_hosts_ls(...))`

Reply:
done